apps/frontend/src/components/Landing.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import Nav from "./Nav";
+import { Navigate, useNavigate } from "react-router-dom";
+
+type Props = {};
+
+const Landing = (props: Props) => {
+  const navigate = useNavigate();
+
+  const handleLoginClick = () => {
+    navigate("/login");
+  };
+  return (
+    <div>
+      <Nav onLoginClick={handleLoginClick} />
+    </div>
```

Comment:
- see above for `handleLoginClick`.
- Aside from that, and this applies to all jsx returns in the PR: You can return `Nav` directly without the `div` since you only have 1 element for now, but if you happen to have multiple elements, it's best to wrap them in `React.Fragments` instead of a `div`, unless you want a wrapper to apply CSS to.
```suggestion
    <>
      <Nav />
      <p>I'm a second element</p>
    </>
```

Reply:
It is done